pyproject.toml

```
@@ -38,6 +38,13 @@ dev = [
     "pytest-cov",
 ]
 
+[project.scripts]
+ppafm-generate-elff = "ppafm.cli.GenerateELFF:main"
+ppafm-generate-elff-point-charges = "ppafm.cli.GenerateELFF_point_charges:main"
+ppafm-generate-ljff = "ppafm.cli.GenerateLJFF:main"
```

These have typos. Doesn't work on Linux filesystems which are case-sensitive:
```suggestion
ppafm-generate-elff = "ppafm.cli.generateElFF:main"
ppafm-generate-elff-point-charges = "ppafm.cli.generateElFF_point_charges:main"
ppafm-generate-ljff = "ppafm.cli.generateLJFF:main"
```